message_definitions/v1.0/ras_a.xml

```diff
@@ -423,7 +423,7 @@
       <field type="float" name="vel_d" units="m/s" invalid="NaN">Down velocity of the POI. NAN if unknown.</field>
       <field type="float" name="hdg" units="rad" invalid="NaN">Heading of the POI in the NED frame. NAN if unknown.</field>
     </message>
-    <message id="450" name="EXPLORATION_STATUS">
```

Comment:
These are in the range for common.xml (and development.xml while being reviewed). We will be using this range in future.